app/controllers/Management.scala

```diff
@@ -22,22 +23,22 @@ import play.api.libs.ws.WSClient
 import scalaz.{Semigroup, Validation, ValidationNel}
 import scalaz.syntax.std.option._
 
-class Management @Inject()(override val wsClient: WSClient)  extends Controller with LazyLogging with OAuthActions {
+class Management(override val wsClient: WSClient, actorSystem: ActorSystem, fBackendFactory: TouchpointBackends)  extends Controller with LazyLogging with OAuthActions {
```

Comment:
the @inject annotation is for runt ime DI which we aren't using in general.  So I have removed this as the controller is explicitly created now rather than relying on the framework to find out how to create one